### PR TITLE
feat: add jobs cleaner

### PR DESCRIPTION
**Pull request by cindyliu-tec**
add a job resource cleaner, when a job's status.completedTime field has value, consider the job is completed, is safe to delete

### CODE REVIEW

**Review by cindyliu-tec — COMMENTED** [review]
reviewed by myself

**Comment by yonahd** [comment]
Amazing contribution!
Overall looks great. I'll do a final review after you make the small adjustments to pass gating

File: pkg/kor/jobs.go [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+		}
+		// checks if the resource's age (measured from its last modified time) matches the included criteria
+		// specified by the filter options.
+		if included, _ := HasIncludedAge(job.CreationTimestamp, filterOpts); !included {
+			continue
+		}
+
+		// if the job has completionTime and succeeded count greater than zero, think the job is completed
+		if job.Status.CompletionTime != nil && job.Status.Succeeded > 0 {
+			jobNames = append(jobNames, job.Name)
```
Comment (yonahd) [inline pkg/kor/jobs.go:39]: Rename to `unusedJobNames`

Comment (cindyliu-tec) [reply pkg/kor/jobs.go]: done

**Pull request merged**